lib/user_input.rb

```diff
@@ -1,24 +1,11 @@
 require_relative './messages'
-require_relative './game_commands'
+require_relative './commands'
 
 module UserInput
-  include GameCommands, Messages
+  include Commands, Messages
 
   def get_upcase_input
     guess_prompt
```

Comment:
It is unclear where guess_prompt comes from. The other comments in this review cover why, but do you see that it non-trivial to determine that `guess_prompt` is a method which comes from the `Messages` module? 

Being able to call `Messages.guess_prompt` or better yet `Logger.game(Messages.guess_prompt)` is a big improvement in clarity.

This also raises the point that there is a concrete dependency on `puts` in the messages module.

Reply:
Should each of my modules be classes instead? I had decided to use modules because none of them have/need constructors, but perhaps that's not a good way to determine if something should be a class vs. module.